modules/developers-guide/examples/rest/curl_insert_list_data.sh

```diff
@@ -0,0 +1,9 @@
+curl -X POST "http://localhost:8082/v2/keyspaces/users_keyspace/users" \
+-H  "accept: application/json" \
+-H  "X-Cassandra-Token: 48bcfab2-b1c6-44fd-a56f-9f2221930096" \
+-H  "Content-Type: application/json" \
+-d "{\"firstname\": \"Janesha\",
+  \"lastname\": \"Doesha\",
+  \"favorite color\": \"grey\",
+  \"top_three_tv_shows\": \"[ 'The Magicians', 'The Librarians', 'Agents of SHIELD' ]\"
```

Comment:
This is the legacy format for the v1 API: the CQL literal in a JSON string. It's still perfectly valid with v2, however we now also accept more JSON-friendly formats:

* a JSON array for list, set or tuple columns:
    ```
    \"top_three_tv_shows\": ["The Magicians", "The Librarians", "Agents of SHIELD"]
    \"favorite_books\": ["Emma", "The Color Purple"]
    \"current_country\":  ["France", "2016-01-01", "2020-02-02"]
     ```
* a JSON array of entry objects for map columns:
    ```
    \"evaluations\": [ {\"key\": "2020", \"value\": "good"}, {\"key\": "2019", \"value\": "okay"} ]
    ```
* a JSON object for UDT columns:
    ```
    { \"street\": \"1, main st\", \"zip\": 12345 }
    ```

Reply:
OK, I'll test again. I put in what worked for me in cURL and postman. Thanks for the info.

Reply:
Note to self to investigate for map (per Madhavan):
"affiliation": [{"key" : "key", "value": "value"}]

Reply:
@olim7t I tried and got this error:
curl -s -X POST "http://localhost:8082/v2/keyspaces/users_keyspace/users" \
> -H  "X-Cassandra-Token: $AUTH_TOKEN" \
> -H  "Content-Type: application/json" \
> -H  "Accept: application/json" \
> -d "{\"firstname\": \"Janesha\",
>   \"lastname\": \"Doesha\",
>   \"favorite color\": \"grey\",
>   \"top_three_tv_shows\": ["The Magicians", "The Librarians", "Agents of SHIELD"]
> }"

{"description":"Server error: Unrecognized token 'The': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')\n at [Source: (String)\"{\"firstname\": \"Janesha\",\n  \"lastname\": \"Doesha\",\n  \"favorite color\": \"grey\",\n  \"top_three_tv_shows\": [The\"; line: 4, column: 29]","code":500}

Reply:
per Olivier, all the double quotes in the array must be escaped:
`curl -s -X POST "http://localhost:8082/v2/keyspaces/users_keyspace/users" \
-H  "X-Cassandra-Token: $AUTH_TOKEN" \
-H  "Content-Type: application/json" \
-H  "Accept: application/json" \
-d "{\"firstname\": \"Janesha\",
  \"lastname\": \"Doesha\",
  \"favorite color\": \"grey\",
  \"top_three_tv_shows\": [ \"The Magicians\", \"The Librarians\", \"Agents of SHIELD\" ]
}"`